src/main/java/org/stellar/sdk/responses/MuxedAccount.java

```diff
@@ -0,0 +1,43 @@
+package org.stellar.sdk.responses;
+
+import com.google.common.base.Objects;
+
+public class MuxedAccount {
+  private final String muxedAddress;
+  private final String accountIdAddress;
```

Comment:
This is the G-Address, right? I would probably call it something like unmuxedAddressed. I am not sure of `accountId` being self-explanatory in this context.

Otherwise, a comment would do.